src/models/User.js

```diff
@@ -16,6 +16,9 @@ export default (sequelize, DataTypes) => {
     password: {
       type: DataTypes.STRING,
       allowNull: false
+    },
+    isVerified: {
+      type: DataTypes.BOOLEAN,
```

Comment:
Why is the default value missing in the model?

Reply:
Okay. I will insert it now.

Reply:
The default value is still missing